test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,31 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() throws Exception{
+        // empty list
+        assertNotNull();
+
+        // 1 null items
+        assertAnyNull((Object) null);
+
+        // some null items
+        assertAnyNull(null, null);
+        assertAnyNull(null, 1,null);
+        assertAnyNull('a', null, "a", null);
+
+        // no null items
+        assertNotNull("a", "b");
+        assertNotNull(1, 2, 3);
+        assertNotNull(1, "");
+    }
+
+    private void assertAnyNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
```

Comment:
Not necessary to abstract the `assertTrue(Utils.isAnyNull(...))` as separate method, because asserting the result of the method itself is fine.

Although existing test `assertAreUnique` does that, but that is to prevent `Arrays.asList(...)` from flying everywhere.